### PR TITLE
Merge changes from master into 2.0.0

### PR DESCRIPTION
**Pull request by darrellwarde**
# Description

As per title.

### CODE REVIEW

File: packages/graphql/package.json [hunk 1]
```diff
@@ -41,7 +41,7 @@
         "@types/faker": "5.1.7",
         "@types/is-uuid": "1.0.0",
         "@types/jest": "26.0.8",
-        "@types/jsonwebtoken": "8.5.0",
+        "@types/jsonwebtoken": "^8.5.4",
```
Comment (danstarns) [inline packages/graphql/package.json:44]: This is the only one that's variable 🤔

Comment (darrellwarde) [reply packages/graphql/package.json:44]: We're a bit all over the place with this, our actual `dependencies` have it, but our `devDependencies` are locked - these could probably all also be prefixed with `^`!

Comment (danstarns) [reply packages/graphql/package.json:44]: Let's not change anything seeming it works just fine, I fear adding a ^ could introduce unforeseen issues.

**Review by danstarns — APPROVED** [review]
MERGE it 🥳

**Pull request merged**